docs/pages/admin-guides/infrastructure-as-code/teleport-operator/secret-lookup.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Looking up values from secrets
+description: How to store sensitive values in a Kubernetes Secret and have the operator look them up.
+---
+
+## How it works
+
+Some Teleport resources might contain sensitive values. Since Teleport 16.4, select CR fields can reference an existing
+Kubernetes secret and the operator will retrieve the value from the secret when reconciling.
+
+Currently only the GithubConnector and OIDCConnector `client_secret` field support secret lookup.
+
+## Prerequisites
+
+To follow this guide you need:
+
+- A running Teleport cluster
+- [A functional Teleport Kubernetes Operator setup](../teleport-operator.mdx#setting-up-the-operator)
```

Comment:
We should be consistent with how we capitalize "Operator". Sometimes it's capitalized on this page and sometimes it's in lowercase.